### PR TITLE
Fixes a mesh with wedge cells

### PR DESCRIPTION
**Pull request by bishtgautam**
- Fixes the computation of subface area for wedge cell
- Updates prefix of existing command-line options 
- Adds a command-line option to specify an initial condition file and
  modifies an existing test to use the new command-line option

### CODE REVIEW

**Comment by bishtgautam** [comment]
This PR will be rebased after #168 is merged.

File: src/tdycore.c [hunk 1]
```diff
@@ -559,6 +563,10 @@ PetscErrorCode TDySetFromOptions(TDy tdy) {
   ierr = PetscOptionsEnd(); CHKERRQ(ierr);
   tdy->setupflags |= TDyOptionsSet;
 
+  if (tdy->init_from_file && tdy->init_with_random_field) {
+    SETERRQ(PETSC_COMM_WORLD,PETSC_ERR_USER,"Both -tdy_init_from_file and -tdy_init_with_random_field cannot be specified");
```
Comment (jeff-cohere) [inline src/tdycore.c:567]: Does `SETERRQ` automatically raise an error, or do we need to call `CHKERRQ` here to get the error message and halt the program?

**Review by jeff-cohere — APPROVED** [review]

**Comment by jeff-cohere** [comment]
@bishtgautam, just confirming: this PR is ready to go, right? No longer a WIP?

**Comment by bishtgautam** [comment]
I'm working on adding a test for wedge mesh.

File: src/tdycore.c (hunk 1 above)
Comment (bishtgautam) [reply src/tdycore.c:567]: No need to call `CHKERRQ` see https://www.mcs.anl.gov/petsc/petsc-current/docs/manualpages/Sys/SETERRQ.html

**Comment by bishtgautam** [comment]
@jeff-cohere There is already a test for wedge mesh. So, this is ready to be integrated.

**Pull request merged**